<launch>
	<env name="ROSCONSOLE_FORMAT" value="[${severity}] [${walltime}] [${node}]: ${message}"/>

<group if="$(arg bag)">
	<node pkg="arcore"
				type="nomarker_position_estimator"
				name="nomarker_position_estimator"
				output="screen"
				args="">
				<remap from="arcore_camera" 						to="optar/arcore_camera_bag" />
				<remap from="kinect_camera" 					to="kinect01/qhd/image_mono_bag" />
	</node>
	
</group>
<group unless="$(arg bag)">	
	<node pkg="arcore"
				type="nomarker_position_estimator"
				name="nomarker_position_estimator"
				output="screen"
				args="">
				<remap from="arcore_camera" 						to="optar/arcore_camera" />
				<remap from="kinect_camera" 					to="kinect01/qhd/image_mono" />
	</node>
</group>

</launch>